src/zod/personal-info.schema.ts

```diff
@@ -0,0 +1,14 @@
+import { z, type ZodType } from 'zod';
+import { type ProfileFormData } from '#/modules/users/types';
+
+export const PersonalInfoSchema: ZodType<Partial<ProfileFormData>> = z.object({
+  firstName: z
+    .string()
+    .min(1, { message: 'Please enter a valid first name' })
```

Comment:
Change the message to be more specific. "Please enter your first name" would make more sense since if length is 0, it means the input field is still empty.

Reply:
fixed